src/stateless/FilterChip.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div :class="{ 'filter-chip__active': active, 'filter-chip__open': isOpen }" class="filter-chip" @click="click()">
```

Comment:
`active` and `open` are modifiers, so you should use `--` instead of `__`. You can also use `prefix` filter: `{ active: active, open: isOpen } | prefix('filter-chip--')`